site/docs/v1/tech/shared/_user-search-limits-workarounds.adoc

```diff
@@ -4,3 +4,9 @@ For example, if you needed to download all users, you could query for users with
 
 If you have access to the Elasticsearch server, you may also run your query directly against it. The Elasticsearch query result will return Elasticsearch document Ids. Each document Id corresponds to a FusionAuth user Id. To retrieve user data, which is most likely what you are after, use the link:/docs/v1/tech/apis/users#search-for-users[User Search API] with one or more user Ids.
 
+[NOTE.since]
```

Comment:
I would say put this note at the top of the section (this file) so that devs on later versions see that first. The `nextResults` route seems like the better option to me. 

Maybe update the verbiage of the note to indicate that it is a better option because it doesn't require segmenting the users to perform the queries. I think it would still be considered a workaround though, right?

Reply:
Or possibly split this into two subsections and put a note at the top of one of them that it's only since 1.48?